features/mesh/dataplanes/Index.feature

```diff
@@ -78,4 +70,3 @@ Feature: mesh / dataplanes / index
       | Value        |
       | fake-backend |
       | http         |
-      | Feb 18, 2021 |
```

Comment:
Has someone asked for this to be removed from the table? I ask because, if you remember in some of our convos, I'm very much "a use case for tables/summary views is to easily compare" and I can imagine a use case where someone might want to know which dataplane was most recently updated.

Reply:
This wasn’t requested. However, I noticed that on small screens, the DPP list in particular is too cramped and I wanted to free up some room. I don’t think "Last updated" is very important and I think it’s the least important column in the table.

Also, with the addition of the "Go to details" entry, the table gains more content, so it becomes doubly relevant.

Reply:
Hmm, not sure we should probably discuss this more widely, I think at the very least it should be in a separate PR.

Overall, we can remove as many columns as we like, but the dataplane names and service names are pretty likely to be longer than we have and therefore push the content out further than we could ever keep up with with just removing more columns.

I kinda think we should split the rows into two separate lines. The dataplane name being on the top, and other information being on the bottom, but thats a whole other story.

---

Umm whilst this is pretty small it ideally should be in another PR so its clear we've done it and not hidden amongst this very large run of PRs for adding summary views. But that being said I'm gonna resolve this.